docs/cloud/tcld/account/metrics/accepted-client-ca/add.md

```diff
@@ -0,0 +1,73 @@
+---
+id: add
+title: tcld account metrics accepted-client-ca add
+sidebar_label: add
+description: How to add client CA certificates to the metrics endpoint of a Temporal Cloud account using tcld.
+tags:
+  - reference
+  - tcld
+---
+
+The `tcld account metrics accepted-client-ca add` command adds client CA certificates to the metrics endpoint of a Temporal Cloud account.
```

Comment:
[Nit] Idk what language we are using elsewhere, but swapping the order of the words and using "client certificate CA"  makes more sense. If we are using this same wording in tcld, will repair that in a follow-up PR

Reply:
In https://docs.temporal.io/cloud/how-to-manage-certificates-in-temporal-cloud, we distinguish between CA certificates and end-entity certificates. Because a certificate must chain up to the CA cert of the account, it'd be more accurate to characterize the endpoint certs as end-entity certificates, right?